### PR TITLE
Add icons/links into README

### PR DESCRIPTION
**Pull request by jedie**
It's helpful to have a direct link to PyPi and the icons are nice, isn't it?

Looks like:

grafik

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # APsystems EZ1 -  Python Library
 
+[![apsystems-ez1 @ PyPi](https://img.shields.io/pypi/v/energymeter2mqtt?label=apsystems-ez1%20%40%20PyPi)](https://pypi.org/project/apsystems-ez1/)
+[![License GPL-3.0-or-later](https://img.shields.io/pypi/l/energymeter2mqtt)](https://github.com/SonnenladenGmbH/APsystems-EZ1-API/blob/main/LICENSE)
+
```
Comment (mawoka-myblock) [inline README.md:5]: Why does it mention energymeter2mqtt in the shields source link?

**Review by mawoka-myblock — CHANGES_REQUESTED** [review]

**Comment by mawoka-myblock** [comment]
That's a good idea and thanks for the effort!

File: README.md (hunk 1 above)
Comment (jedie) [reply README.md:5]: Oh, sorry. Copy&paste error. I missed to replace the string.

Comment (jedie) [reply README.md:5]: fixed

**Review by mawoka-myblock — APPROVED** [review]

**Comment by mawoka-myblock** [comment]
Thank you! Und Grüße nach Duisburg!

**Pull request merged**